platform/default/default_file_source.cpp

```diff
@@ -312,12 +312,12 @@ void DefaultFileSource::setMaximumCacheSize(uint64_t cacheSize) const {
 }
 
 void DefaultFileSource::pause() {
-    thread->pause();
+    //thread->pause();
     priorityThread->pause();
 }
 
 void DefaultFileSource::resume() {
-    thread->resume();
+    //thread->resume();
     priorityThread->resume();
 }
```

Comment:
priorityThread only processes immediate requests for tile display on the map. MapBox added the pause/resume functionality specifically so that wouldn't make these requests in the background. It also makes requests to 'revalidate' tiles that are already on screen. 

thread processes adding and deleting regions, eviction, saving tiles to the database, fetching tiles for download, and fetching tiles for the watch. I think it is fine running in the background so long as we aren't using any file protection. This also allows the downloads to continue for 5 minutes after closing the app.